collections/sea-ice-index.yaml

```diff
@@ -72,6 +72,14 @@ Extent:
         - -90
         - 180
         - 90
+      - - -180
```

Comment:
Nitpicking since the stac json file is generated correctly. 
I would format this the same as above (new line for the lonely `-` and then the bbox coordinates after that)?

Also, should the "original" bbox above be removed?

FFTR.

Reply:
Formatted accordingly to the other bboxes.

> should the "original" bbox above be removed?

No, the "rule" here is if there is several bboxes the 1st one is one covering all single ones and afterwards the single ones follow. 

> not sure where 39.23 and -30.98 come from, but it looks ok. It could also be 0 (for both), so each would be exactly half of the original bbox

It's mentioned in the `GeographicalCoverage` above.

Reply:
> No, the "rule" here is if there is several bboxes the 1st one is one covering all single ones and afterwards the single ones follow.

That makes sense. Thanks.

> It's mentioned in the GeographicalCoverage above.

Ah true, thanks.